app/editor/src/upload-context/interfaces/IUploadFile.ts

```diff
@@ -0,0 +1,8 @@
+import { FetchState } from './FetchState';
+
+export interface IUploadFile {
```

Comment:
This interface name appears to be a little misleading.  It seems to be `IUploadState` would reflect what it is better.